### ISSUE TITLE
what does the ball mod do?

### ISSUE THREAD

**Issue by Sanityfrog502** [eosuwu/Gorilla-Tag-Public-Mods#1]

**Comment by Sanityfrog502** [comment]
also non of the mods work :(

**Comment by shibbylol** [comment]
Hello, ball mod was just some testing of mine. It doesn't do anything.
The mods should work, are you sure you are using them correctly?

**Comment by yupo-netizen** [comment]
yeah the mods dont work because of the update i think, the only one i found to work was the dash monke
